lib/tree.js

```diff
@@ -261,8 +261,13 @@ var tree = function(path, options) {
   var totalSize = 0
   var directorySize = getDirectorySize(options)
 
-  return paths(path, options)
-  .map(function(f) {
+  var pathsArray = paths(path, options)
+
+  if(Array.isArray(path)) {
+   path = ''
+  }
+
+  return pathsArray.map(function(f) {
```

Comment:
This change is bothering me and should not be needed. 

Take a look at the `paths` function [here](https://github.com/bkanuka/explorer/blob/99fa1097cce217f015d1d6db530399c6fb7c1968/lib/tree.js#L200). 

Your custom function is leading to the path not being an array nor a string there. However, the data returned by the custom searched should be parsed [here](https://github.com/bkanuka/explorer/blob/99fa1097cce217f015d1d6db530399c6fb7c1968/lib/search/index.js#L15). 
It the output is how you said:

```
somepath
somepath2
```

The search will output an array and this should not be needed.